### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by shin-**

### CODE REVIEW

File: tests/integration/__init__.py [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-# Remove imports when merged in master and Jenkins is updated to find the
-# tests in the new location.
-from .api_test import *
-from .build_test import *
-from .container_test import *
-from .exec_test import *
-from .image_test import *
-from .network_test import *
-from .regression_test import *
-from .volume_test import *
```
Comment (dnephin) [inline tests/integration/__init__.py]: Don't you still need an empty __init__.py here for the tests to be considered python modules?

Comment (shin-) [inline tests/integration/__init__.py]: Yep. It's still there :)

Comment (shin-) [inline tests/integration/__init__.py]: https://github.com/docker/docker-py/blob/remove-ci-transition-files/tests/integration/__init__.py

Comment (dnephin) [inline tests/integration/__init__.py]: Ah, I should have looked at the expanded changed files. cool

**Comment by dnephin** [comment]
LGTM

**Pull request merged**